entries/SWC-124.md

```diff
@@ -0,0 +1,26 @@
+# Title
+Write to Arbitrary Storage Location
+
+## Relationships
+
+[CWE-123: Write-what-where Condition](https://cwe.mitre.org/data/definitions/123.html)
+
+## Description
+
+A smart contract's data (e.g., storing the owner of the contract) is persistently stored
+at some storage location (i.e., a key or address) on the EVM level. The contract is
+responsible for ensuring that only authorized clients may write to sensitive storage
+locations. If an attacker is able to write to arbitrary locations within a contracts

```

Comment:
If an attacker is able to write to arbitrary locations within a contract's storage, the authorization checks may easily be circumvented. This can allow an attacker to corrupt the storage;

Reply:
I've address this.